lib/browser/lighthouse.js

```diff
@@ -138,8 +138,7 @@ function parseLhResult(results) {
       notApplicableObj[id] = auditInfo;
     } else if (scoreDisplayMode !== 'manual') {
       const snippets = [];
-      if (details && details.items.length) {
-        details.items.forEach(item =>
+      (details && details.items || []).forEach(item =>
```

Comment:
How about defaulting the value to an empty array when destructuring it?

```javascript
const { id, description, score, scoreDisplayMode, details = [] } = audit;
```

Assuming we are guarding against `undefined` only and don't care about `null` 😉